addons/skin.estuary/xml/Variables.xml

```diff
@@ -473,6 +473,11 @@
 		<value condition="!String.IsEmpty(ListItem.Premiered) + ListItem.IsNew">[COLOR grey]$LOCALIZE[20416]:[/COLOR] $INFO[ListItem.Premiered] [B]([COLOR button_focus]$LOCALIZE[842][/COLOR])[/B][CR]</value>
 		<value>$INFO[ListItem.Premiered,[COLOR grey]$LOCALIZE[20416]:[/COLOR] ,[CR]]</value>
 	</variable>
+	<variable name="RecordingDateSizeLabel">
+		<value condition="!String.IsEmpty(ListItem.Date) + !String.IsEmpty(ListItem.Size)">$INFO[ListItem.Date,[COLOR grey],[/COLOR]]$INFO[ListItem.Size, (,)][CR]</value>
+		<value condition="!String.IsEmpty(ListItem.Size)">$INFO[ListItem.Date,[COLOR grey],[/COLOR]][CR]</value>
+		<value>$INFO[ListItem.Date,[COLOR grey],[/COLOR][CR]]</value>
+	</variable>
```

Comment:
i'm not really following those conditions, especially the second one..
perhaps i'm wrong but couldn't it be simplified to:
```
<variable name="RecordingDateSizeLabel">
	<value condition="!String.IsEmpty(ListItem.Size)">$INFO[ListItem.Date,[COLOR grey],[/COLOR]]$INFO[ListItem.Size, (,)[CR]]</value>
	<value>$INFO[ListItem.Date,[COLOR grey],[/COLOR][CR]]</value>
</variable>
```

Reply:
Sorry, yes you are correct ;)